packages/lib-react-components/src/Modal/Modal.js

```diff
@@ -67,13 +67,41 @@ class Modal extends React.Component {
   }
 }
 
+Modal.defaultProps = {
+  active: false,
+  className: '',
+  closeFn: undefined,
+  headingBackground: 'brand',
+  title: '',
+  titleColor: 'neutral-6'
+}
```

Comment:
We should either be using `defaultProps` here or the destructured defaults in the component, but not both. It's confusing and unclear which takes precedence. FWIW, default props are going to be deprecated by React for functional components (https://github.com/facebook/react/pull/16210).

Reply:
Let's try default arguments and see if the storybook docs display them.

Reply:
Ok, with default arguments, the 'Default' column is empty:
![Screenshot of the Modal Docs page with nothing in the default values column for arguments.](https://user-images.githubusercontent.com/59547/105831187-53d7ab00-5fbe-11eb-87c9-6a2558a8c923.png)

Reply:
With `Modal.defaultProps`, the defaults are documented.
![Screenshot of the Modal Docs page, showing default prop values in the arguments table.](https://user-images.githubusercontent.com/59547/105831634-deb8a580-5fbe-11eb-96a4-0db07821a9d0.png)

Reply:
I think storybook will document default args for functions, or `Component.defaultProps` for  class components, but it can't parse component code to read destructured props for classes.

From the link you posted above, [`Component.defaultProps` will be around forever for classes](https://github.com/facebook/react/pull/16210#discussion_r307420058), so I'm going to use that here.